src/shared/models/transactions/stacks-transaction.model.ts

```diff
@@ -1,7 +1,7 @@
 import type { MempoolTransaction, Transaction } from '@stacks/stacks-blockchain-api-types';
 
 export type StacksTx = MempoolTransaction | Transaction;
-export type StacksTxStatus = 'failed' | 'pending' | 'success_anchor_block' | 'success_microblock';
```

Comment:
Sure on this change? Can't we see these still?

Reply:
I can add this back to be safe, but was thinking if microblocks aren't active a transaction wouldn't ever be in a microblock?

Reply:
When I look in the api lib I don't see that those tx statuses exist any longer?

<img width="864" alt="Screenshot 2024-01-30 at 4 22 22 PM" src="https://github.com/leather-wallet/extension/assets/6493321/8cdb39a8-6234-4ce4-af0d-f6298a6185a4">

<img width="389" alt="Screenshot 2024-01-30 at 4 22 39 PM" src="https://github.com/leather-wallet/extension/assets/6493321/b541372a-ca52-4537-b516-dc78eaa49cb1">

Reply:
pending def exists right? when its in mempool

Reply:
Nvm, I see we added those on our end...

<img width="593" alt="Screenshot 2024-01-31 at 9 50 47 AM" src="https://github.com/leather-wallet/extension/assets/6493321/d66b1c08-2387-43b2-a841-2ae97da8ea01">

Reply:
Yeah, I think this is ok to remove bc we weren't really using it anywhere. As long as we keep `success` we should be good.

Reply:
I didn't remove `pending` btw, think you aren't looking at the change correctly?

Reply:
I just changed to, but did not remove completely...

<img width="534" alt="Screenshot 2024-01-31 at 10 15 51 AM" src="https://github.com/leather-wallet/extension/assets/6493321/fd16c4f7-df8a-4b6a-9ed3-3a6b778d8c94">